lib/pysquared/battery_helper.py

```diff
@@ -275,14 +275,16 @@ def debug_timing(self):
                 values = [float(x) for x in parts[:4]]
                 values.append(bool(int(parts[4])))
             except Exception as e:
-                print(f"Parse error: {e}")
+                self.logger.error(message=f"Parse error: {e}")
         parse_time = (time.monotonic() - parse_start) * 1000
 
         # Total time
         total_time = (time.monotonic() - start) * 1000
 
-        print(f"Send time: {send_time:.2f}ms")
-        print(f"Read time: {read_time:.2f}ms")
-        print(f"Parse time: {parse_time:.2f}ms")
-        print(f"Total time: {total_time:.2f}ms")
-        print(f"Response: {response}")
+        self.logger.info(
+            send_time=f"{send_time:.2f}ms",
+            read_time=f"{read_time:.2f}ms",
+            parse_time=f"{parse_time:.2f}ms",
+            total_time=f"{total_time:.2f}ms",
+            response=f"{response}",
+        )
```

Comment:
Missing a message. What about something like this?
```suggestion
        self.logger.info(
            "timing analysis",
            send_time=f"{send_time:.2f}ms",
            read_time=f"{read_time:.2f}ms",
            parse_time=f"{parse_time:.2f}ms",
            total_time=f"{total_time:.2f}ms",
            response=f"{response}",
        )
```